packages/engine-frontend/components/IntegrationSearch.tsx

```diff
@@ -58,42 +69,46 @@ export function IntegrationSearch({
           <Loader className="size-5 animate-spin text-[#8A5DF6]" />
         </div>
       ) : (
-        <div className="grid max-h-[388px] grid-cols-1 justify-items-center gap-4 overflow-y-auto py-4 sm:grid-cols-2 md:grid-cols-3">
-          {ints?.map((int) => (
-            <WithConnectorConnect
-              key={int.id}
-              connectorConfig={{
-                id: int.connector_config_id,
-                connector: int.ccfg.connector,
-              }}
-              onEvent={(e) => {
-                onEvent?.({
-                  type: e.type,
-                  integration: {
-                    connectorConfigId: int.connector_config_id,
-                    id: int.id,
-                  },
-                })
-              }}>
-              {({openConnect}) => (
-                <Card
-                  className={cn(
-                    'flex w-[150px] cursor-pointer flex-col items-center gap-2 rounded-lg p-4',
-                    'focus:outline-none focus:ring-2 focus:ring-indigo-600',
-                    'transition-transform duration-200 ease-in-out hover:scale-105',
-                  )}
-                  onClick={() => openConnect()}>
-                  <ConnectorLogo
-                    connector={int.ccfg.connector}
-                    className="size-[80px] rounded-lg"
-                  />
-                  <span className="mt-2 text-center text-sm font-bold text-muted-foreground">
-                    {int.name}
-                  </span>
-                </Card>
-              )}
-            </WithConnectorConnect>
-          ))}
+        <div className="space-y-6 overflow-y-auto py-4">
+          {Object.entries(intsByCategory ?? {}).map(
+            ([category, categoryInts]) => (
+              <div key={category}>
+                <h3 className="mb-2 text-lg font-semibold">
+                  {category
```

Comment:
Can we make this that if the length of this is < 5 characters without spaces (I.e. HRIS) to put it in caps? That way we show ATS and not Ats.

Reply:
Done in the last commit